test/functional/inspec_exec_test.rb

```diff
@@ -1306,15 +1306,27 @@ def stderr
     end
   end
 
-  describe "when profiles are dependent on different versions of same profile" do
+  describe "when profiles are dependent on different versions of same profile - test in unix" do
     let(:profile) { "#{profile_path}/git-fetcher/inheritance/parent-profile" }
     let(:run_result) { run_inspec_process("exec #{profile}") }
     it "should evaluate all test controls of all versions correctly" do
+      skip_windows!
       _(run_result.stderr).must_be_empty
       _(run_result.stdout).must_include "2.7.0"
       _(run_result.stdout).must_include "2.6.0"
       _(run_result.stdout).must_include "sshd-01"
       _(run_result.stdout).must_include "sshd-50"
     end
   end
+
+  describe "when profiles are dependent on different versions of same profile - test in windows" do
+    let(:profile) { "#{profile_path}/git-fetcher/inheritance-windows/parent-profile" }
+    let(:run_result) { run_inspec_process("exec #{profile}") }
+    it "should evaluate all test controls of all versions correctly" do
+      if is_windows?
```

Comment:
Minor complaint - maybe move this outside the describe so we don't run the test at all on unix?